package com.ExceptionHandling.Throw;

public class Demo2 {
	
	public static void main(String[] args) {
	
		try 
		{
		System.out.println("hello");		
		}
		catch (ArithmeticException exception) 
		//catch (Exception exception)
		//catch (IOException exception)
		//catch(InterruptedException exception)
		//catch (Error error)
		{
			
		}
	}
}

